flexget/plugins/output/symlink.py

```diff
@@ -27,6 +29,8 @@ class Symlink:
         ]
     }
 
+    destination_field = 'link_to'
```

Comment:
very weird place for a constant in python, and we don't generally have constants for entry fields

please inline this variable

Reply:
I have based my changes following move code, I felft wierd about it but the same thing it does in move code. The constant is not realy necessary here I deleted it.